src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -709,6 +732,8 @@ private static Options buildOptions() {
         options.addOption(OPTION_C_NAME, true, "Sets the check configuration file to use.");
         options.addOption(OPTION_O_NAME, true, "Sets the output file. Defaults to stdout");
         options.addOption(OPTION_P_NAME, true, "Loads the properties file");
+        options.addOption(OPTION_S_NAME, true, "Generates basic suppression xpath");
+        options.addOption(OPTION_TAB_WIDTH_NAME, true, "Sets the length of the tab character");
```

Comment:
Just like the `f` option, we should print the default.

Reply:
He meant , mention in message.

Reply:
Done